Index: config.inc.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/config.inc.php b/config.inc.php
new file mode 100644
--- /dev/null	(date 1675342685889)
+++ b/config.inc.php	(date 1675342685889)
@@ -0,0 +1,68 @@
+<?php
+/**
+ * @ignore 配置文件
+ * @ignore 默认情况请勿随意修改数据，敏感数据请勿随意复制给别人
+ *
+ * @author 筱锋xiao_lfeng
+ * @license MIT
+ * @copyright 2022-2023 ZCW and YN. All Rights Reserved.
+ * @version 1.0.0 2023-01-16更新
+ */
+
+// 格式化数据
+$config = [];
+
+/******************* 开放管理 *******************/
+// 站点启用
+$config['WEB']['Start'] = TRUE;
+// 站点开启 Debug 模式
+$config['WEB']['DeBUG'] = FALSE;
+// 站点通信密钥
+$config['SESSION'] = "YNily20040227";
+
+// AccessKeyID
+$config['Aliyun']['AccessKeyID'] = 'LTAI4FyjsXWYDqgTSHrMrCha';
+// AccessKeySecret
+$config['Aliyun']['AccessKeySecret'] = 'DYmFXkMu7ZMly7mzd1M7UYbbYd4UH4';
+
+/******************** 数据库 ********************/
+// 数据库地址
+$config['SQL']['host'] = '127.0.0.1';
+// 数据库名字
+$config['SQL']['dbname'] = 'nw_love';
+// 数据库用户名
+$config['SQL']['username'] = 'nw_love';
+// 数据库密码
+$config['SQL']['password'] = 'YNily20040227';
+
+/******************** 数据表 ********************/
+// 网站基本信息数据表
+$config['TABLE']['info'] = 'nw_web_info';
+// 睡觉信息统计数据表
+$config['TABLE']['daily'] = 'nw_daily';
+// 相册列表数据库
+$config['TABLE']['AlbumList'] = 'nw_album_list';
+// 相册内数据表
+$config['TABLE']['Album'] = 'nw_album';
+// 相册别人偷拍
+$config['TABLE']['candid'] = 'nw_other_candid';
+
+/******************* 邮箱管理 *******************/
+// 邮箱地址
+$config['SMTP']['HOST'] = 'smtp.qiye.aliyun.com';
+// 是否允许SMTP认证
+$config['SMTP']['AUTH'] = true;
+// 邮箱用户名（邮箱）
+$config['SMTP']['USER'] = 'noreplay@x-lf.cn';
+// 邮箱密码或授权码
+$config['SMTP']['PASSWORD'] = 'X+7ily20040722';
+// 发件人
+$config['SMTP']['NAME'] = 'noreplay@x-lf.cn';
+// 展示名字
+$config['SMTP']['DISPLAY'] = 'noreplay@x-lf.cn';
+// 收件名字
+$config['SMTP']['REPLAY'] = 'gm@x-lf.cn';
+// 非SSL端口
+$config['SMTP']['NOSSL'] = 25;
+// SSL端口
+$config['SMTP']['SSL'] = 465;
\ No newline at end of file
